### PR TITLE
Fixes #7295: slapd core dumps on 1TB RAM

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/7295

### CODE REVIEW

File: rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init [hunk 1]
```diff
@@ -390,6 +390,10 @@ start_slapd() {
       MEMSIZE=$(sed -n '/^MemTotal/s/[^0-9]//gp' /proc/meminfo)
       # Round down using integer division (800 bytes is the size of an entry)
       CACHESIZE=$((${MEMSIZE} * 1024 / 800 / 10 / 10000 * 10000))
+      # Maximum cache size of 40M entries. Above about 40G of RAM (about 53.6M entries) OpenLDAP dumps core, so we force an upper limit.
```
Comment (peckpeck) [inline rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init]: 40M entries is about 30G of RAM to be coherent with the 800 bytes estimation above.

**Comment by jooooooon** [comment]
PR updated

File: rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init (hunk 1 above)
Comment (jooooooon) [inline rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init]: > 40M entries is about 30G of RAM to be coherent with the 800 bytes
> estimation above.
>

Woops. I put the dot in the wrong place :)

Comment (peckpeck) [inline rudder-inventory-ldap/SOURCES/rudder-inventory-ldap.init]: The dot was at the right place, I just misread the comment

**Comment by jooooooon** [comment]
PR updated

**Comment by peckpeck** [comment]
OK merging

**Pull request merged**